medmodels/statistic_evaluations/statistical_analysis/inferential_statistics.pyi

```diff
@@ -0,0 +1,24 @@
+from typing import List, Tuple
+
+from numpy.typing import ArrayLike
+
+from medmodels.statistic_evaluations.comparer.data_comparer import TestSummary
+
+def test_for_normal_distribution(sample: ArrayLike) -> bool: ...
+def decide_hypothesis_test(samples: List[ArrayLike], alpha: float) -> TestSummary: ...
+def two_tailed_t_test(samples: List[ArrayLike], alpha: float) -> TestSummary: ...
+def mann_whitney_u_test(samples: List[ArrayLike], alpha: float) -> TestSummary: ...
+def test_ANOVA(samples: List[ArrayLike], alpha: float) -> TestSummary: ...
```

Comment:
We should not have caps or test_ as the start of a function (it will be included in the make test)

Reply:
ah true :D this one is killing me, usually you would write it in caps :D